### PR TITLE
A try to fix huge_tree for lxml for issue #24

### PR DESCRIPTION
**Pull request by falkowich**
A try to fix issue https://github.com/greenbone/gvm-tools/issues/24
I don't know if this helps out, or not.

This is my first try at a pull request..
If it's messy and you hate it, I can always take it as a learning experience :)

--
Regards Falk

### CODE REVIEW

**Comment by falkowich** [comment]
@wiegandm, true bout the security restrictions.
It feels somewhat counterproductive to introduce a security vuln. in gmv-tools :)

I'll look into if i can add attribute for that. At least a mockup that you can nice up later? :)

--
Regards Falk

**Comment by wiegandm** [comment]
> I'll look into if i can add attribute for that

I add `raw_response` in greenbone/gvm-tools@ff2658af34c51203a00dfe008d4c6adf7a2ce278, might be useful as a guideline.

**Comment by falkowich** [comment]
Humm, I was lost in my own thougts with this is seems like.  

We don't need a huge_tree option. It looks like that what's needed was only a parser option added to the `tree = etree.parse(f, parser)`

It's the `recover=True` option in etree.XMLParser that is the pill for this error.
From [http://lxml.de/parsing.html](http://lxml.de/parsing.html): recover - try hard to parse through broken XML

So this pull only adds one line of code.

--
Regards Falk

**Comment by falkowich** [comment]
What do you think, is this possible to merge?

I use it in "production" and it works[tm] as intended :)
